layouts/partials/home/partners.html

```diff
@@ -16,10 +16,10 @@ <h2 class="mb-3">Our Community</h2>
             <div class="partner-logo lightspeedstudios-logo"></div>
             <div class="partner-logo microsoft-logo"></div>
             <div class="partner-logo niantic-logo"></div>
+            <div class="partner-logo epicgames-logo"></div>
```

Comment:
Please add this in the right position (between AWS and Huawei). Partners of the same category are listed in alphabetical order.

Reply:
+1. Agreed!